tests/tas-elasticsearch/src/test/java/org/alfresco/elasticsearch/ElasticsearchLiveIndexingTests.java

```diff
@@ -102,7 +103,7 @@ public void dataPreparation()
         dataUser.addUserToSite(userMultiSite, siteModel1, UserRole.SiteContributor);
         dataUser.addUserToSite(userMultiSite, siteModel2, UserRole.SiteContributor);
 
-        createContent(FILE_0_NAME, "This is the first test containing the unique word " + UNIQUE_WORD, siteModel1, userSite1);
+        file0 = createContent(FILE_0_NAME, "This is the first test containing the unique word " + UNIQUE_WORD, siteModel1, userSite1);
```

Comment:
Is this variable ever used in the test class?

Reply:
It's not any more - thanks!